src/tests/components/Signup.test.js

```diff
@@ -0,0 +1,24 @@
+import { describe } from "@jest/globals";
+import React from "react";
+// import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
+import { Router } from "react-router-dom";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import Signup from "../../Views/Signup";
+
+describe("Signup", () => {
+  const initialState = { signup: { loading: false, error: "", response: "" } };
+  const mockStore = configureStore();
+  let store;
+
+  it("Should render signup page", () => {
+    store = mockStore(initialState);
+    let wrapper = shallow(
+      <Provider store={store}>
+        <Signup />
+      </Provider>
+    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
please avoiding using toMatchSnapshot, [read this for reference ](https://medium.com/@tomgold_48918/why-i-stopped-using-snapshot-testing-with-jest-3279fe41ffb2)